infra/tpu-pytorch-releases/artifacts_builds.tf

```diff
@@ -170,7 +171,7 @@ module "versioned_builds" {
   for_each = local.versioned_builds_dict
 
   ansible_vars = merge(each.value, {
-    pytorch_git_rev = each.value.git_tag
+    pytorch_git_rev = each.value.pytorch_git_tag
```

Comment:
Should this default to `each.value.git_tag` since `pytorch_git_tag` is not set for all versioned builds?